videointelligence/annotate/text_detection_gcs.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package annotate contains speech examples.
+package annotate
+
+// [START videointelligence_text_detection_gcs]
+
+import (
+	"fmt"
+	"io"
+	"log"
+
+	"context"
+
+	"github.com/golang/protobuf/ptypes"
+
+	video "cloud.google.com/go/videointelligence/apiv1"
+	videopb "google.golang.org/genproto/googleapis/cloud/videointelligence/v1"
+)
+
+func textDetectionGCS(w io.Writer, gcsURI string) error {
+	// gcsURI := "gs://python-docs-samples-tests/video/googlework_short.mp4"
```

Comment:
Might want a fake path here, in case we ever need to change it?

Reply:
I kind of like something a user can plug-and-play and test really quickly that something is happening, rather than making them find a file. If this path breaks down at some point, it will break down the test too since it's using the same file so it should be easy to fix.